README.md

```diff
@@ -85,7 +85,12 @@ By default, messages will be sent to a SQS service running on `localhost:4576`.
 
 ### AWS SQS mode
 
-Use this mode by setting `LAMBDA_WRAPPER_OFFLINE_SQS_MODE=aws`. Messages will be sent to the real queue in AWS. This is useful for running end-to-end tests where a message is sent to a queue and eventually appears in an external data store.
+Use this mode by setting `LAMBDA_WRAPPER_OFFLINE_SQS_MODE=aws`. Messages will be sent to the real queue in AWS. This mode is useful when a queue is consumed by an external service, rather than another function in the service under test.
```

Comment:
We don't use `dotenv` for this project ... at least, I can't see it in the packaga.json and neither is there a `.env.sample` file. So the environment variables have to be set manually? Can you add a note to this effect for clarity.

Reply:
These variables are to be set in the project that's using `@comicrelief/lambda-wrapper`, not in `lambda-wrapper` itself.

Reply:
What about for local development and testing of this project?

Reply:
As discussed, `lambda-wrapper` is a library rather than an application. We're only unit-testing it, and for these features we're setting up the env vars as part of the tests: https://github.com/comicrelief/lambda-wrapper/pull/567/files#diff-ebabdc49b10517d848389670bd2a31b24d0ec894ab3ff855c4209723e1990e87R173